shared/chat/conversation/messages/message-popup/payment/container.tsx

```diff
@@ -237,7 +246,7 @@ const RequestPaymentPopup = Container.connect(
   requestMapStateToProps,
   requestMapDispatchToProps,
   requestMergeProps
-)(PaymentPopup)
+)(PaymentPopup) as any
```

Comment:
What happened here?

Reply:
the type was complicated and i didn't want to fix it up short term